river/View.zig

```diff
@@ -506,6 +514,12 @@ pub fn notifyAppId(self: Self) void {
     }
 }
 
+/// Called when the view requests to be focused
+pub fn requestActicate(self: *Self) void {
+    self.current.urgent = true;
```

Comment:
Same thing here, we must modify the pending state not the current state (and in this case start a transaction to apply it).

Also this function is only called in handleRequestActivate(), why don't we just inline this code there and get rid of this function?